packages/website/src/components/Playground.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-unnecessary-condition */
```

Comment:
ditto were there a lot here?

Reply:
Yeah. They should be improved by https://github.com/typescript-eslint/typescript-eslint/issues/7630. I'll mark that here.